config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/Entity.java

```diff
@@ -96,17 +96,17 @@ public String getName() {
                 return name;
             }
 
+            @JsonIgnore
```

Comment:
This will conflict with `definedEntities`. There we need `id` and `guid`.

Reply:
@nikhilahuja i have tested that, no conflict